src/deadline/unreal_adaptor/UnrealAdaptor/adaptor.py

```diff
@@ -280,7 +280,14 @@ def _start_unreal_client(self) -> None:
 
         unreal_exe = "UnrealEditor-Cmd"
         unreal_project_path = self.init_data.get("project_path", "")
-        extra_cmd_str = self.init_data.get("extra_cmd_args", "")
+
+        # First, read args from file since it can be too long to pass
+        # them to Job parameter (1024 chars limit)
+        extra_cmd_str = ""
+        extra_cmd_args_file = self.init_data.get("extra_cmd_args_file", "")
```

Comment:
Will this change break backwards compatibility? 

Wouldn't customers that currently use `extra_cmd_args` field now fail to work?
Or did it never work as the command string were always greater than 1024 characters?

Reply:
This change shouldn't break backwards compatibility because this parameter suppose to be optional and added if only any parameters passed
Also we remove `extra_cmd_args` and use only `extra_cmd_args_file` that always include inherited cmd args from User's Unreal session